### PR TITLE
add allowBlank option to number validator

### PR DESCRIPTION
**Pull request by jakesjews**

### CODE REVIEW

File: app/validators/number.js [hunk 1]
```diff
@@ -12,6 +12,10 @@ const {
 
 export default Base.extend({
   validate(value, options) {
+    if (options.allowBlank && isEmpty(value)) {
```
Comment (offirgolan) [inline app/validators/number.js]: To be consistent with the rest of the validators, can you move this block below the variable declaration

**Comment by offirgolan** [comment]
Ah I totally forgot about this when I wrote this one. Thanks for the PR :D. Besides my note, can you also add the allowBlank options in the docs (docs/docs/validators/number.md). You can just copy it from one of the other ones.

**Comment by jakesjews** [comment]
No problem I'll get that in real quick

**Comment by offirgolan** [comment]
Looks good. Thank you :smile_cat:

**Pull request merged**